packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -269,6 +270,11 @@ export const messages = {
   auth_custom_label: 'Custom',
   auth_custom_detail: 'Enter a custom login URL',
   auth_invalid_url: 'URL must begin with http:// or https://',
+  auth_logout_scratch_prompt:
```

Comment:
@sbudhirajadoc Prompt and options for scratch org confirmation dialog.

Reply:
can I see a screen of this confirmation dialog please?

Reply:
@sbudhirajadoc I updated the description with an image of the confirmation dialog.